plutarch-test/goldens/trace.dev=false.ptrace.uplc.eval.golden

```diff
@@ -0,0 +1,2 @@
+one (program 1.0.0 ())
```

Comment:
How do you decide which tests to make versions of for both `dev=true` and `dev=false`?

Reply:
Answered elsewhere, and explained in README.